### DISCUSSION TITLE
How to find a row by userId (relationship)

### DISCUSSION THREAD

**Discussion by Mathie01** [prisma/prisma#13200] [Q&A]
Hello !

I have this model : 

```
model Response {
  id       Int      @id @default(autoincrement())
  survey   Survey   @relation(fields: [surveyId], references: [id])
  surveyId Int
  user     User     @relation(fields: [userId], references: [id])
  userId   Int
  sendAt   DateTime @default(now())
  json     Json
}
```

I would like to do a query to find the row that has a userID equal to 2 and a surveyId equal to 3.

Except that I feel like I can't just put the number as a parameter.

How can I do it?

**Comment by binajmen** [comment]
Sure you can 😃 

```ts
await prisma.response.findUnique({
  where: { userId: 2, surveyId: 3 }
})
```
See https://www.prisma.io/docs/concepts/components/prisma-client/filtering-and-sorting.

If you expect to have one and only one `response` per `survey` per `user`, you can use `findUnique()` instead, and you should probably define an unique constraint `@@unique([surveyId, userId])` to enforce that. 

See https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#unique-1.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
